docs/components/board/pca9685.md

```diff
@@ -76,7 +79,17 @@ The following attributes are available for `pca9685` boards:
 <!-- prettier-ignore -->
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `board_name` | string | **Required** | The `name` of the board with GPIO pins your `pca9685` is [connected to](https://learn.adafruit.com/16-channel-pwm-servo-driver/hooking-it-up). |
+| `board_name` | string | Optional | The `name` of the board with GPIO pins your `pca9685` is [connected to](https://learn.adafruit.com/16-channel-pwm-servo-driver/hooking-it-up). |
+| `i2c_name` | string | Optional | The `name` of the `i2c` your `pca9685` is connected to, as configured in your main board's section of the config file. |
```

Comment:
```suggestion
| `i2c_name` | string | Optional | The `name` of the `i2c` your `pca9685` is connected to, as configured in your main board's JSON config. |
```
(opt) another way to phrase it that might be simpler!

Reply:
I think if “section” makes sense to you I’ll leave it as-is because it’s all in the same config, just different bits!